crates/bevy_render/src/extract_component.rs

```diff
@@ -197,18 +195,7 @@ impl<C: ExtractComponent> Plugin for ExtractComponentPlugin<C> {
     }
 }
 
-impl<T: Asset> ExtractComponent for Handle<T> {
-    type QueryData = Read<Handle<T>>;
-    type QueryFilter = ();
-    type Out = Handle<T>;
-
-    #[inline]
-    fn extract_component(handle: QueryItem<'_, Self::QueryData>) -> Option<Self::Out> {
-        Some(handle.clone_weak())
-    }
-}
-
-/// This system extracts all components of the corresponding [`ExtractComponent`], for entities that are synced via [`crate::sync_world::SyncToRenderWorld`].
+/// This system extracts all components of the corresponding [`ExtractComponent`], for entities that are synced via [`SyncToRenderWorld`].
```

Comment:
This is making CI unhappy
```suggestion
/// This system extracts all components of the corresponding [`ExtractComponent`], for entities that are synced via [`SyncToRenderWorld`](crate::sync_world::SyncToRenderWorld).
```

Reply:
I botched the merge a bit :(